packages/microsoft-security-adapter/src/constants/entra.ts

```diff
@@ -18,16 +18,21 @@ export const SERVICE_BASE_URL = 'https://entra.microsoft.com'
 export const APP_ROLE_ASSIGNMENT_FIELD_NAME = 'appRoleAssignments'
 export const APP_ROLES_FIELD_NAME = 'appRoles'
 export const AUTHENTICATION_METHOD_CONFIGURATIONS_FIELD_NAME = 'authenticationMethodConfigurations'
+export const AUTHENTICATION_STRENGTH_FIELD_NAME = 'authenticationStrength'
 export const CUSTOM_SECURITY_ATTRIBUTE_ALLOWED_VALUES_FIELD_NAME = 'allowedValues'
 export const DELEGATED_PERMISSION_CLASSIFICATIONS_FIELD_NAME = 'delegatedPermissionClassifications'
 export const DOMAIN_NAME_REFERENCES_FIELD_NAME = 'domainNameReferences'
+export const GRANT_CONTROLS_FIELD_NAME = 'grantControls'
```

Comment:
Looks like it shouldn't be exported? (same for the other consts)
```suggestion
const GRANT_CONTROLS_FIELD_NAME = 'grantControls'
```
also a nit, if it's only used in `AUTHENTICATION_STRENGTH_PATH` there's no really a need to put in const

Reply:
You're right, it's a leftover since at first I didn't share the path, thanks! :)